composer.json

```diff
@@ -118,5 +119,7 @@
             "url": "https://github.com/alanpoulain/FOSUserBundle",
             "no-api": true
         }
-    ]
+    ],
+    "minimum-stability": "dev",
```

Comment:
These changes are unnecessary. Just use `^4.3@dev`

Reply:
Currently, the are, or I'll have to add some extra deps to composer.json: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpClient/composer.json#L23-L24

Reply:
I think that should be preferable to using these root-only options.